packages/web-runtime/src/container/bootstrap.ts

```diff
@@ -253,6 +253,24 @@ export const announceClientService = ({
   vue.prototype.$clientService.owncloudSdk = sdk
 }
 
+/**
+ * announce uppyService and inject it into vue
+ *
+ * @param vue
+ * @param store
+ */
+export const announcePermissionManager = ({
+  vue,
+  store
+}: {
+  vue: VueConstructor
+  store: Store<any>
+}): void => {
+  const permissionManager = new PermissionManager(store)
+  vue.prototype.$permissionManager = permissionManager
```

Comment:
You don't need to do both. Either set it on `prototype` or use reactive `set`. They both have the same result, so one is sufficient.

Reply:
I tried that.
If I only use set(vue, '$permissionManager', permissionManager), the permissionManager is accessible in /home/jan/working/web/packages/web-app-user-management/src/index.js but not in 
/home/jan/working/web/packages/web-app-files/src/views/spaces/Projects.vue

and vice versa, I have no clue why this is the case :x